mods/ossl/init.lua

```diff
@@ -41,6 +41,19 @@ if not ossl.registered then
 		Then, we'll have him shot.
 	--]]
 
+	local env = minetest.request_insecure_environment()
+	if not env then
+		minetest.log("error", "[ossl] Failed to get an insecure environment. " ..
+			"Please add this mod to the trusted mods list in the server settings.")
+		asset(env)
```

Comment:
assert()  ?

Reply:
Meant to crash the whole thing and stop the loading... Couldn't find another way to do that.